harmony-api/src/routes/post/following/followingRouter.ts

```diff
@@ -4,6 +4,9 @@ import * as followingController from '../../../controllers/post/following/follow
 
 const router = express.Router()
 
-router.get('/', followingController.getFollowingFeed)
+router.get('/', followingController.getAllFollowingFeed)
+router.get('/user', followingController.getFollowingUserFeed)
```

Comment:
Are these other following feeds actually getting used on the frontend? If not can we remove them (and their corresponding controller/service/whatever functions)?

Reply:
I was thinking we could add a filter to the following feed so you can see posts from only followed users, songs, albums, or artists

Reply:
I think thats a good idea, although I think the endpoint they hit shod be constant, and the filters should be applied through query parameters, and we would let the service and repo layers apply the logic to make the filters happen. I think we could reduce some code duplication this way and simplify some frontend logic.